Makefile

```diff
@@ -102,8 +102,11 @@ dynamic_properties:
 unit_tests:
 	./gradlew test $(log_level)
 
+gradle_test:
+	./gradlew gradleTest $(log_level)
+
```

Comment:
I'm thinking of a common task for unit and gradle tests by a common filter.

Case: 
I have a feature in a plugin. It is covered by unit tests and gradle tests:
- my_plugin/scr/gradleTest/feature/...
- my_plugin/scr/test/feature/...

I made a change in feature and want to run corresponding tests but not all. I found only this way:

`./gradlew :subprojects:gradle:my-plugin:test --tests "feature.*" :subprojects:gradle:my-plugin:gradleTest --tests "feature.*"`

For some reason, `--tests` applies only for one task.

What do you think about this case? Am I missing something?